Bottom: 8fda8cc1f044c1e25a06ce33b756cb1ba4005dbd
Top:    9d47ac621319f399eadcef0ddc1208377dad79d3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-12 01:39:40 +0100

Win32: Select the correct GL libraries to link against


---

diff --git a/configure.ac b/configure.ac
index 74d6e9e..cd696a2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -429,7 +429,7 @@ Either do not use --enable-gl or enable the gtk HID.])
 	esac
 
 	GLU_CFLAGS=
-	GLU_LIBS=-lGLU
+	GLU_LIBS=-lglu32 -lopengl32
 
 	AC_DEFINE([ENABLE_GL], 1,
 		[Define to 1 if GL support is to be compiled in])
